src/utilities.cpp

```diff
@@ -431,6 +432,15 @@ bool isStandardUnitName(const std::string &name)
     return standardUnitsList.count(name) != 0;
 }
 
+bool isStandardUnit(const UnitsPtr &units)
+{
+    bool is = false;
+    if (units != nullptr) {
+        is = units->isBaseUnit() && isStandardUnitName(units->name());
+    }
+    return is; //standardUnitsList.count(name) != 0;
```

Comment:
Why that comment? I would delete it.

Having said that, I would rewrite the whole method to simply read:

```
bool isStandardUnit(const UnitsPtr &units)
{
    if ((units != nullptr) && units->isBaseUnit() && isStandardUnitName(units->name())) {
        return true;
    }

    return false;
}
```